action.yaml

```
@@ -43,7 +43,13 @@ runs:
       env:
         GITHUB_TOKEN: ${{ github.token }}
       id: pkgcheck
+    - id: check_visnet
+      run: |
+        if [ -f "${{ steps.pkgcheck.outputs.visnet_path }}" ]; then
+          echo "exists=true" >> $GITHUB_OUTPUT
+        fi
     - uses: actions/upload-artifact@v4
+      if: steps.check_visnet.outputs.exists == 'true'
       with:
         name: visual-network
```

```suggestion
      with:
        if-no-files-found: ignore
        name: visual-network
```
This would be an alternative, it prevents the action from failing if the files isn't found.

---

Oh, that's much easier. I'll update tomorrow, and remove the extra step. Or feel free to do it yourself if you've got time. Thanks!